elevation_mapping/include/elevation_mapping/sensor_processors/PerfectSensorProcessor.hpp

```diff
@@ -56,10 +56,10 @@ class PerfectSensorProcessor : public SensorProcessorBase
    * @param[out] variances the elevation map height variances.
    * @return true if successful.
    */
-   virtual bool computeVariances(
-		   const pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr pointCloud,
-		   const Eigen::Matrix<double, 6, 6>& robotPoseCovariance,
-		   Eigen::VectorXf& variances);
+  virtual bool computeVariances(
+    const pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr pointCloud,
```

Comment:
Sorry, I think my description might not have been accurate enough.
All arguments need 2 more whitespaces.
Definitely last request. Thank you very much!